eva-accession-clustering-automation/clustering_automation/cluster_from_mongo.py

```diff
@@ -92,15 +92,15 @@ def generate_linear_pipeline(taxonomy_id, scientific_name, assembly_list, common
             command_to_run=f'java -Xmx{memory}m -jar {clustering_artifact} --spring.config.location=file:{properties_path} '
                            f'--spring.batch.job.names=CLUSTER_UNCLUSTERED_VARIANTS_JOB',
             process_directives={'memory': process_directives_for_java_pipelines['memory'],
-                                'clusterOptions': process_directives_for_java_pipelines['clusterOptions'] +
-                                                  ' -g /accession/{instance:02d} '} # needed to serialize accessioning
+                                'clusterOptions': f"{process_directives_for_java_pipelines['clusterOptions']}"
+                                                  f" -g /accession/{instance} "}  # needed to serialize accessioning
         ))
         pipeline.add_process(
             process_name=f'end_{suffix}',
             command_to_run=status_update_template.format(status='Completed')  # TODO: how to choose completed/failed?
         )
         # TODO add QA process
-    return pipeline
+    return pipeline, output_directory
```

Comment:
The `output_directory` is different for every [assembly](https://github.com/EBIvariation/eva-accession/pull/310/files#diff-f5d03ec235616a63adf93f3730b453bd9e57bba3743c657ec38025e1a3e9302fL60) so we would be returning the output directory for the last assembly only but the linear pipeline might include multiple assemblies. is that ok?

Reply:
Good point. I changed this to the parent directory which is the species